packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -287,6 +293,17 @@ export class CodeModelBuilder {
     }); // include all versions and do the filter by ourselves
     this.program.reportDiagnostics(this.sdkContext.diagnostics);
 
+    // license
+    if (this.sdkContext.sdkPackage.licenseInfo) {
+      this.codeModel.info.license = new License(this.sdkContext.sdkPackage.licenseInfo.name, {
+        url: this.sdkContext.sdkPackage.licenseInfo.link,
+        extensions: {
+          header: this.sdkContext.sdkPackage.licenseInfo.header,
+          company: this.sdkContext.sdkPackage.licenseInfo.company,
+        },
+      });
+    }
```

Comment:
Still using m4 info.

`name` and `link` maps to `licenseInfo`.
`header` maps to `license-header` in JavaSettings.
`company` currently is not used. (it could be used to populate `developer` section in pom.xml, but I think this is not an exact map)